webkit2/cl-webkit2.asd

```diff
@@ -48,7 +48,11 @@
                (:file "webkit2.print-operation")
                (:file "webkit2.script-world")
                (:file "webkit2.frame")
+               (:file "webkitdom.dom-object")
                (:file "jscore.js-value-ref")
+               (:file "jscore.jsc-context")
+               (:file "jscore.jsc-class")
+               (:file "jscore.js-value")
```

Comment:
Still js-value?  I guess it's a typo!  Make sure to test from a fresh image :)

Reply:
Fixed!